benchmark/worker/messageport.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const common = require('../common.js');
+const { MessageChannel } = require('worker_threads');
+const bench = common.createBenchmark(main, {
+  payload: ['string', 'object'],
+  n: [1e6]
+});
+
+function main(conf) {
+
+  const n = +conf.n;
```

Comment:
I don't think we usually cast `n` to a number, why is this needed here?

Reply:
I don’t know, it’s copied from the other test

Reply:
So based on the code in `benchmark/common.js` and specifically `_parseArgs` it should detect that `n` is a number in the config and accordingly cast the parameter to a number, so this line shouldn't be needed.

Reply:
Sure, done – removed the coercion